tensorrt_llm/bench/benchmark/serve.py

```
@@ -0,0 +1,256 @@
+    kv_cache_percent = params.pop("kv_cache_free_gpu_mem_fraction")
+    beam_width = params.pop("max_beam_width")
+    enable_chunked_context: bool = params.pop("enable_chunked_context")
+    scheduler_policy: str = params.pop("scheduler_policy")
+
+    # Update configuration with runtime options
+    exec_settings["settings_config"]["kv_cache_percent"] = kv_cache_percent
+    exec_settings["settings_config"]["max_batch_size"] = runtime_max_bs
+    exec_settings["settings_config"]["max_num_tokens"] = runtime_max_tokens
+    exec_settings["settings_config"]["beam_width"] = beam_width
+    exec_settings["settings_config"][
+        "scheduler_policy"] = CapacitySchedulerPolicy.GUARANTEED_NO_EVICT if scheduler_policy == "guaranteed_no_evict" else CapacitySchedulerPolicy.MAX_UTILIZATION
```

Ruff (0.12.2)

Line too long (163 > 120)

(E501)